README.md

```diff
@@ -1,18 +1,10 @@
 # Ably OpenAPI specifications
 
-This repository contains an OAS3 specification for the [Ably REST API](https://ably.com/documentation/rest-api). In the future, additional specifications will be added here.
+This repository contains Ably OAS3 specifications.
 
-## Ably client library SDKs
+## Viewing the specification locally
 
-Ably provides official client library SDKs developed by Ably that offer a consistent yet idiomatic API across every language. Ably recommends that customers use our comprehensive native SDKs to consume our REST APIs. Download an SDK at [ably.com/download](https://ably.com/download).
-
-If you wish to access the raw REST API however, Ably does provide an OpenAPI and Swagger compatible specification in this repository in [swagger.yaml](./swagger.yaml).
-
-## Viewing the specification
-
-The specification is published online at [https://api-docs.ably.com/docs/service-api](https://api-docs.ably.com/docs/service-api).
-
-In order to view the specification locally, run `nmp install` and `npm run start`, and open the page [localhost:3000/api-docs](http://localhost:3000/api-docs) in your browser.
+In order to view the specification locally, run `http-server -c-1`, and point your browser at `localhost:8080`.
```

Comment:
Why did you remove the installation instructions, `npm install`?

Reply:
After an `npm install` I could not run `http-server -c-1` anyway.  I think the instructions on how to use this need to be updated.  Are you missing the defintion in the package.json for redoc?